test/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner_test.rb

```diff
@@ -125,35 +125,52 @@ def self.it_triggers_compilation_process
   describe ".install_dependencies" do
```

Comment:
Since `.check_compilation_dependencies!` is public and used outside of `install_dependencies`, we should make sure it has its own tests too.